_posts/blog/2022-12-1-vector-library-vs-vector-database.md

```diff
@@ -85,14 +83,14 @@ The table below summarizes the differences between vector libraries and database
 | Incremental importing, concurrent reading while importing| No (some do, e.g. hnswlib) | Yes |
 | Stores objects and vectors | No | Yes|
 |Speed | Typically faster than full-blown database|  Typically slower than pure library|
-| Performance optimized for | In-memory similarity search| End2end-callstack, including: vector search, object retrieval from persistent storage, optional inverted index filtering, network requests, etc. |
+| Performance optimized for | In-memory similarity search| Full functionality, including: vector search, object retrieval from persistent storage, optional inverted index filtering, etc. |
 | Durability, crash recovery | No | Yes |
 | Persistence | Only at explicit snapshot | Immediate (after each insert, update or delete) |
-| Sharding (i.e. distribute dataset among more than one machine, automatically merge/aggregate results at query time across machines) | No (there are some 3rd-party projects that wrap sharding around existing libraries) | Yes |
+| Sharding (i.e. distribute datasets among more than one machine, automatically merge/aggregate results at query time across machines) | No (there are some 3rd-party projects that wrap sharding around existing libraries) | Yes |
```

Comment:
Maybe change this to "distribute a dataset ..."

Reply:
Done!